apps/src/sites/studio/pages/courses/show.js

```diff
@@ -1,30 +1,48 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import CourseOverview from '@cdo/apps/templates/courseOverview/CourseOverview';
-import { ViewType } from '@cdo/apps/code-studio/stageLockRedux';
+import { setViewType, ViewType } from '@cdo/apps/code-studio/stageLockRedux';
+import { getStore } from '@cdo/apps/code-studio/redux';
```

Comment:
This is an entry point but I don't see a `registerReducers` call.  Where does that happen for this page?

Reply:
We only need the common code-studio reducers (at least so far).

Reply:
I didn't even realize [code-studio/redux.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/redux.js) existed!  Or that it's registering reducers when it's loaded on this line - my brain just assumed `@cdo/apps/redux` when I was that.  That's kind of scary subtle.

Reply:
A fair point. Some of this subtlety is also what got me into trouble when accidentally removing teacherSections from some places I shouldnt have.